### PR TITLE
Court list

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
Hello again! :) In this PR I started with the CSS of the CourtCardsList page. The page is now scrollable and the data from the local storage is inside the white background. The positioning of the <p> on each card will be difficult! (different lengths of names). My goal is, that all the list cards are looking exactly like the add court card at the AddCourt page. if the names or locations are getting longer, all positionings are gone for now. I need to do it next time! :)

thank u for ur review! :)

### CODE REVIEW

**Review by soerendolling — APPROVED** [review]
Looks all fine to me

File: src/components/CourtCards.css [hunk 1]
```diff
@@ -25,23 +32,23 @@
...
+.addBasketballIcon {
   margin-right: 10px;
   margin-top: 15px;
 }
 
-.courtName {
+.addCourtName {
   font-size: 14px;
   margin: 15px;
 }
```
Comment (yvlange) [inline src/components/CourtCards.css:49]: Right now there is a big gap between entering the court name and the location. I think there is where the rating should be placed in right?

File: src/components/CourtCards.css [hunk 2]
```diff
@@ -69,7 +76,7 @@
   box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
 }
 
-.courtDescription {
+.addCourtDescription {
   overflow-wrap: break-word;
   padding-left: 10px;
 }
```
Comment (yvlange) [inline src/components/CourtCards.css:82]: Maybe place the input fields either left or in the middle. Right now the Upload Photo area and the description field is centered but location and opening times is left.

File: src/components/CourtCardsList.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.listCourtInformation {
+  margin: 10px 70px 10px 13px;
+}
+
+.listInformationText {
+  margin-left: 20px;
+  font-size: 12px;
+}
```
Comment (yvlange) [inline src/components/CourtCardsList.css:49]: Still think neumorphism would look amazing in your app 😍  
https://neumorphism.io/#e0e0e0

File: src/components/CourtCardsList.js [hunk 4]
```diff
@@ -1,9 +1,58 @@
...
+
+            <div className="listGroupCourt">
+              <p className=" listTextField">{court.description}</p>
+            </div>
+          </div>
+        </section>
+      );
+    });
+  }
+  return <div>{renderCourt()}</div>;
```
Comment (yvlange) [inline src/components/CourtCardsList.js:55]: All the responsiveness seems to work when I tried it. Great job!! :)

**Review by yvlange — APPROVED** [review]
So far amazing work @Alexander-Schulte 🎉  just some tiny suggestions :)

**Pull request merged**